### PR TITLE
Replaced Getter/Setters with Lombok Annotations.

### PR DESCRIPTION
**Pull request by avinashkris9**
Used @Data to replace Getter/Setter/Constructor.
Used lombok specific annotations on all other cases
Refactored QueueService/UserService functions to align with the new constructor parameter ordering.

### CODE REVIEW

File: complete/src/main/java/com/example/restservice/model/QueueDetailsResponse.java [hunk 1]
```diff
@@ -4,110 +4,36 @@
...
+import lombok.Data;
+import lombok.Getter;
+import lombok.Setter;
+
+@AllArgsConstructor
+@Getter
+@Setter
 public class QueueDetailsResponse {
+
+  @Getter@Setter@AllArgsConstructor
```
Comment (daltonfury42) [inline complete/src/main/java/com/example/restservice/model/QueueDetailsResponse.java:17]: Can you put this on separate lines?

File: complete/src/main/java/com/example/restservice/model/QueueDetailsRequest.java [hunk 2]
```diff
@@ -1,5 +1,10 @@
 package com.example.restservice.model;
 
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
 public class QueueDetailsRequest {
     String queueId;
```
Comment (daltonfury42) [inline complete/src/main/java/com/example/restservice/model/QueueDetailsRequest.java:10]: Looks like you forgot to remove the getters and setters here.

File: complete/src/main/java/com/example/restservice/service/UserService.java [hunk 3]
```diff
@@ -21,12 +21,12 @@
...
         tokenId,
         userRepository
-            .findById(tokenId)
-            .orElseThrow(SQInvalidRequestException::userNotFoundException)
-            .getStatus(),
-        getAheadCount(tokenId).orElseThrow(SQInternalServerException::new));
+                .findById(tokenId)
+                .orElseThrow(SQInvalidRequestException::userNotFoundException)
+                .getStatus());
   }
```
Comment (daltonfury42) [inline complete/src/main/java/com/example/restservice/service/UserService.java:30]: Here and above, I feel that the previous ordering makes more sense. Can you rearrange the member definitions in the classes to restore the constructor ordering?

File: complete/src/main/java/com/example/restservice/model/QueueDetailsResponse.java (hunk 1 above)
Comment (avinashkris9) [reply complete/src/main/java/com/example/restservice/model/QueueDetailsResponse.java]: Updated on latest commit

File: complete/src/main/java/com/example/restservice/service/UserService.java (hunk 3 above)
Comment (avinashkris9) [reply complete/src/main/java/com/example/restservice/service/UserService.java]: restored ordering on latest commit

**Pull request merged**